docs/usage.md

```diff
@@ -12,21 +17,19 @@ EXTENSIONS = {
 }
 ```
 
-This extension also requires you to set the export location using the `SW_EXPORT_URI` settings.
+This extension also requires you to set the export location using the `SW_EXPORT_URI` settings (check the settings page for different options for exporting).
 
 ```python
 SW_EXPORT_URI = "s3://scrapy-webarchive/"
```

Comment:
Just to check: can we also specify the complete destination here? This reads like the full destination is generated with `SW_EXPORT_URI` as a prefix.

It would be nice if the invoking system can determine the final filename, otherwise it needs to guess the resulting WACZ filename afterwards. This can help finding the resulting WACZ later, to have a complete filename already.

Reply:
Good point. I've created a new issue for this, let's pick that separately from this one.